generative_data_prep/processors/article_tokenizer.py

```diff
@@ -141,6 +147,193 @@ def _update_token_metrics(self, tokenized_sequences: List[TokenizedSequence]):
             self.metrics.completion_tokens += seq.completion_tokens
             self.metrics.padding_tokens += seq.pad_tokens
 
+    def process_jsonl_with_chat_template(self, jsonl: List[dict]) -> List[TokenizedArticle]:
```

Comment:
We need to add pytests with unit tests to test this, similar to 

tests/test_article_tokenizer.py

Reply:
Agree, please help to add the unit tests, thanks a lot